src/dlgMapper.cpp

```diff
@@ -329,3 +330,30 @@ void dlgMapper::slot_switchArea(const int index)
     mp2dMap->switchArea(areaName);
 }
 #endif
+
+void dlgMapper::updateInfoContributors() 
+{
+    info_pushButton->menu()->clear();
+    auto* clearAction = new QAction(tr("None"), info_pushButton);
```

Comment:
'none' can be translated quite differently in languages as you know - good idea to give translators the context here:

```suggestion
    auto* clearAction = new QAction(tr("None", "Don't show the map overlay, 'none' meaning no map overlay styled are enabled"), info_pushButton);
```